executor/show.go

```diff
@@ -852,6 +863,9 @@ func ConstructResultOfShowCreateTable(ctx sessionctx.Context, tableInfo *model.T
 			cols = append(cols, colInfo)
 		}
 		fmt.Fprintf(buf, "(%s)", strings.Join(cols, ","))
+		if idxInfo.Invisible {
+			fmt.Fprintf(buf, ` /*!80000 INVISIBLE */`)
```

Comment:
How does comment `80000` come？any reference?

Reply:
Sorry, I do not find any reference for the `/*!80000 INVISIBLE */`,
but when test in MySQL 8.0, it does have this behavior.

Reply:
make sense